### PR TITLE
feat: add row-reverse and column-reverse values to direction property…

### PR DESCRIPTION
**Pull request by DmitriyT51**
… of Container component

[PRI-325]

### Description

Add row-reverse and column-reverse values to direction property of Container component

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [-] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [-] Covered with tests

**Breaking change**

- [-] codemod is created and showcased in the changeset
- [-] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[PRI-324]: https://toptal-core.atlassian.net/browse/PRI-324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PRI-325]: https://toptal-core.atlassian.net/browse/PRI-325?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by DmitriyT51** [comment]
@toptal-anvil ping reviewers

**Review by TomasSlama — APPROVED** [review]
looks good 👍

**Review by mkrl — APPROVED** [review]
Looks great! Thank you. I have rebased the PR and updated one failing snapshot test (not directly related to your PR)

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

**Comment by DmitriyT51** [comment]
> Looks great! Thank you. I have rebased the PR and updated one failing snapshot test (not directly related to your PR)

Thank you @mkrl

**Review by ybouhjira — APPROVED** [review]
Code looks good!

File: packages/picasso/src/Container/Container.tsx [hunk 1]
```diff
@@ -142,6 +142,8 @@ export const Container = documentable(
               [classes.flex]: flex,
               [classes.inline]: inline,
               [classes.column]: direction === 'column',
+              [classes.rowReverse]: direction === 'row-reverse',
+              [classes.columnReverse]: direction === 'column-reverse',
```
Comment (ozgurkececioglu) [inline packages/picasso/src/Container/Container.tsx:146]: I would prefer following the same approach as others for the same property `direction`
```suggestion
              [classes[kebabToCamelCase(direction || '')]]: direction,
```

Comment (DmitriyT51) [reply packages/picasso/src/Container/Container.tsx]: @ozgurkececioglu 
I updated code the way you suggested

**Review by ozgurkececioglu — APPROVED** [review]
tested locally, looking good 👍 
thanks for the update and the contribution 🙇‍♂️

**Pull request merged**